content/src/content/jcr_root/apps/core/wcm/components/button/v2/button/clientlibs/editor/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[core.wcm.components.button.v1.editor]"
```

Comment:
see other comment about category name.

Reply:
@bpauli good catch! Addressed in https://github.com/adobe/aem-core-wcm-components/commit/3b42c20d9f9a3dcc0afb138a7b4432e043467830